### DISCUSSION TITLE
Updated files mark the book as unread/updated

### DISCUSSION THREAD

**Discussion by vonProteus** [gotson/komga#1786] [Ideas]
When a file is updated and the number of pages changes, the book should be marked as `unread` or assigned a new status, such as `updated` or `changed`. If we know that pages were added at the end, we could mark the progress up to the last unchanged page.

**Comment by gotson** [comment]
The current logic only change the read status if the number of pages differ, and the read status is _in progress_. If you have read the book already, upgrading the book should not reset the read status (this was the behaviour before #1093 ).

Could you maybe explain your use case / experience, instead of trying to dictate a solution ?

**Reply by vonProteus** [reply]
Sorry, I didn't mean to dictate a solution; that was not my intention.

I download some web comics, and sometimes the authors don't create new episodes but instead add new pages to existing ones.
As a result, I receive the same file with updated unseen content.

**Reply by gotson** [reply]
this seem like a niche use case to be honest. How do you currently place/update the new file so that Komga sees it ? Are you using the Import feature to upgrade the book ?

**Reply by vonProteus** [reply]
I download directly to the Komga library, and the files are updated in place. After an auto scan of the library, they become available in Komga.
